core/templates/pages/exploration-player-page/exploration-player-page.component.spec.ts

```diff
@@ -136,4 +144,37 @@ describe('Exploration player page', function() {
     expect(elementDescriptionProperty.attr('content')).toBe(
       exploration.objective);
   });
+
+  it('should initialize command executor service',
+    function() {
+      WindowWrapperMessageService.getLocationHref =
+      jasmine.createSpy('parentMessage spy').and.returnValue(
+        'https://www.oppia.org/exploration/embed/fake?' +
+        'secret_hostname=testSecret');
+      CommandExecutorService.initialize = jasmine.createSpy(
+        'outerFrameEvents spy');
+
+      ctrl.$onInit();
+      $scope.$apply();
+
+      expect(CommandExecutorService.initialize)
```

Comment:
This is a little weak -- maybe add isInitialized() and check the value of that is false beforehand and true afterwards.